torch/nn/quantized/modules/conv.py

```diff
@@ -15,8 +15,7 @@
 """Computes the output shape given convolution parameters."""
 def _conv_output_shape(input_size, kernel_size, padding, stride, dilation,
```

Comment:
is this function used in tests only? if so maybe move it to common_quantized.py?